modules/insights-operator-configuring-sca.adoc

```diff
@@ -7,7 +7,7 @@
 [id="insights-operator-configuring-sca_{context}"]
 = Configuring simple content access import interval
 
-You can configure how often the Insights Operator imports the simple content access certificates using the `support` secret in the `openshift-config` namespace. The certificate import normally occurs every 8 hours, but you can shorten this interval if you update your simple content access configuration in Red Hat Subscription Management.
+You can configure how often the Insights Operator imports the simple content access entitlements using the `support` secret in the `openshift-config` namespace. The certificate import normally occurs every 8 hours, but you can shorten this interval if you update your simple content access configuration in Red Hat Subscription Management.
```

Comment:
Spell out "eight".
This is in the [Red Hat Supplemental Style Guide](https://redhat-documentation.github.io/supplementary-style-guide/#_n).